src/control/server/storage_scm_test.go

```diff
@@ -170,6 +170,7 @@ func TestFormatScm(t *testing.T) {
 	tests := []struct {
 		inited    bool
 		formatted bool
+		reformat  bool
```

Comment:
Should there be a new test case where this is true?

Reply:
The tests at this level need to be cleaned up and probably minimized, IMO. They know too much about things happening at lower levels as an artifact of being the only tests for this functionality before the refactor. At this point I think the tests at this level should be primarily focused on making sure that the correct RPC responses happen given various RPC inputs.

There are reformat tests in storage/scm which cover this and other scenarios in more detail.

Reply:
OK with me